src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService user = new UserService();
+        User actualUser = user.findByEmail(email);
+        if (actualUser != null && actualUser.getPassword().equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
You can simplify it when return (condition what inside if) and you don`t need to return true or false

Reply:
@Sanderfuck updated